esx_service/utils/auth_data.py

```diff
@@ -624,6 +688,8 @@ def create_tables(self):
                 vm_id TEXT PRIMARY KEY NOT NULL,
                 -- id in tenants table
                 tenant_id TEXT NOT NULL,
+                -- name of the VM being added to the tenant
+                vm_name TEXT,
                 FOREIGN KEY(tenant_id) REFERENCES tenants(id)
```

Comment:
the order is  vm-uuid, tenant-id, vm-name. For inserts, it takes the order of columns specified in query is used. So doesn't make any difference